src/containerapp/azext_containerapp/custom.py

```diff
@@ -4297,3 +4306,195 @@ def delete_workload_profile(cmd, resource_group_name, env_name, workload_profile
         return r
     except Exception as e:
         handle_raw_exception(e)
+
+
+def patch_list(cmd, resource_group_name=None, managed_env=None, show_all=False):
+    if is_docker_running() is False:
+        logger.warning("Please install or start Docker and try again.")
+        return
+    pack_exec_path = get_pack_exec_path()
```

Comment:
@snehapar9 @harryli0108 -- we should check that `pack_exec_path` isn't `None` and return early if it wasn't installed or found.

Reply:
Updated: added additional check for empty return of `pack_exec_path`